<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="drive_letter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
    <property name="placeholderText">
     <string>Drive letter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="server_addr">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>192.168.0.14:8091</string>
    </property>
    <property name="placeholderText">
     <string>Server address : port</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Error:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cache_folder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>F:\\cache</string>
    </property>
    <property name="placeholderText">
     <string>Drive letter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
